src/components/signup/StepGuide.tsx

```diff
@@ -6,12 +6,9 @@ interface StepGuideProps {
 }
 
 const StepGuide = ({ signupStep }: StepGuideProps) => {
-  // console.log('fromGuide', signupStep);
-console.log('firstStepComplete',firstStepComplete);
-
   return (
```

Comment:
We can make use of components and programming logics to make this page much simpler:
- make components for Steps and resuse them
- Pass step labels and number of steps in props
- Render steps using loop
and write other logics as needed